.travis.yml

```diff
@@ -0,0 +1,90 @@
+language: php
+
+notifications:
+    email:
+        recipients:
+            - dev@dmk-ebusiness.de
+        on_success: change
+        on_failure: always
+
+php:
+    - 5.6
```

Comment:
@hannesbochmann @rengaw83 I think we can drop the PHP 5.6 support? TYPO3 8.7 requires PHP `>=7.0`-